WORKSPACE

```diff
@@ -0,0 +1,41 @@
+workspace(name = "com_github_skypjack_entt")
```

Comment:
This is the workspace name. Anyone who wants to consume an `EnTT` bazel package would prefix it with `@com_github_skypjack_entt//`. This is the standard naming convention. If there is an official EnTT website we could use that instead.